src/components/dock/dock-button/dock-button.tsx

```diff
@@ -107,6 +107,7 @@ export class DockButton {
     private openPopover = (event: MouseEvent) => {
         event.stopPropagation();
         this.isOpen = true;
+        this.itemSelected.emit(this.item);
```

Comment:
This seems to work, in the sense that it fixes https://github.com/Lundalogik/crm-feature/issues/2931, because we happen to have implemented our `itemSelected` event handler in a certain way.

But I don't think it's the right event to emit here. The item is not selected (we don't expect `this.item.selected` to be set to `true`).